skaffold-duct-tape/duct-tape/Dockerfile

```diff
@@ -3,17 +3,27 @@ RUN curl --location --output delve-1.2.0.tar.gz https://github.com/go-delve/delv
  && tar xzf delve-1.2.0.tar.gz
 RUN cd delve-1.2.0/cmd/dlv && go install
 
-#
-#FROM python:2.7 as python27
-#RUN curl --location --output ptsvd.zip https://github.com/Microsoft/ptvsd/archive/v4.2.4.zip \
-# && unzip ptsvd.zip \
-# && cd ptsvd-4.2.4 \
-# && 
+FROM python:2.7 as python27
+RUN curl --location --output ptsvd.zip https://github.com/Microsoft/ptvsd/archive/v4.2.8.zip \
+ && unzip -q ptsvd.zip \
+ && cd ptvsd-4.2.8 \
+ && mkdir -p /ptvsd/lib/python2.7/site-packages \
+ && PYTHONUSERBASE=/ptvsd python setup.py install --user
 
+FROM python:3.7 as python37
+RUN curl --location --output ptsvd.zip https://github.com/Microsoft/ptvsd/archive/v4.2.8.zip \
```

Comment:
Is there a reason why we download and unzip `ptvsd` instead of using `pip install`? It seems we have to update the version link whenever there is a new release if we do it this way?

Reply:
Good point.  I've updated it to use pip.